src/widget/woverview.cpp

```diff
@@ -80,8 +81,8 @@ WOverview::WOverview(
             QStringLiteral("[Waveform]"),
```

Comment:
I really don't understand those `ControlFlag::NoAssertIfMissing` flags above. Some controls are read when painting and there is no null check. Even if default controls return 0 there will a division by 0 at one place.
Should we remove those flags?

Reply:
yeah, so I'll remove those.